roles/pulp_content/README.md

```diff
@@ -8,6 +8,9 @@ Role Variables
 
 * `pulp_content_bind`: Interface and Port where Pulp Content [`gunicorn` service will
   listen.](https://docs.gunicorn.org/en/stable/settings.html#bind)
+* `pulp_content_workers`: Number of `gunicorn` processes for handling Pulp content app requests.
+  Defaults to 8. Used to render the `pulpcore-api.service.j2` template, passing to the `--workers`
+  parameter of the gunicorn service.
```

Comment:
I think the second sentence here is superfluous. The first one documents nicely what the the option is doing for the user. Why do we tell him how we do it?

Reply:
I agree, I'll remove it if @mikedep333 agrees (it'll take 3 hours for the CI to run again so I'll hold off).

Reply:
Well said @mdellweg !